charts/cray-spire/templates/server/workloads.yaml

```diff
@@ -210,6 +218,14 @@ data:
             value: gid:0
           - type: unix
             value: path:/opt/cray/cray-spire/tpm-provisioner
+       - spiffeID: spiffe://shasta/ncn/workload/sbps-marshal
```

Comment:
Here too, `shasta` shouldn't be hardcoded, it should be `{{ .Values.trustDomain }}`